src/mol-plugin-ui/structure/measurements.tsx

```diff
@@ -63,8 +63,15 @@ export class MeasurementList extends PurePluginUIComponent {
     render() {
         const measurements = this.plugin.managers.structure.measurement.state;
 
+        const labels = [];
+        for (const cell of measurements.labels) {
+            const tags = (cell.obj as any)['tags'] as string[];
+            if (!tags || !tags.includes(MeasurementOrderLabelTag))
+                labels.push(cell);
+        }
```

Comment:
I think this filtering should happen when ``structure.measurement.state`` in the manager and not here.

Reply:
True, fixed.